.gitignore

```diff
@@ -158,4 +158,12 @@ cython_debug/
 
 # Image files
 *.JPG
-*.ARW
\ No newline at end of file
+*.ARW
+
+# log files
+mini_computer_api/files_uploaded_to_globus.txt
+mini_computer_api/deleted_files.log
+
+# image folders
+/mini_computer_api/images/*
+/mini_computer_api/test_images/*
```

Comment:
Resolve the conflicts and I don't think we need this directory (/test_images).

Reply:
good catch i will delete that directory and remove from gitignore